config/config.go

```diff
@@ -21,7 +22,8 @@ func NewConfig() *Config {
 	}
 
 	return &Config{
-		Token:          os.Getenv("SC_TOKEN"),
+		Token: os.Getenv("SC_TOKEN"),
+		YadiskToken: os.Getenv("YADISK_TOKEN"),
```

Comment:
It is advisable to follow the rule: the first prefix in the environment is the one that lets us know that the environment belongs to our program, in our case it is SC

Reply:
In other words the environment name must be SC_YADISK_TOKEN